fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/events/p2p-exchange.md

title: P2P exchange
images:
  - src: parade-6.jpg

>     alt: Group of people seated on stacked black plastic crates outdoors in public plaza during sunny day

Reviewer: ```suggestion
    alt: Group of people seated on stacked black plastic crates outdoors among black plastic crate architecture
```

  - src: p2pexchange_a5flyer.jpg

>     alt: Event flyer with blue and green text reading "P2P EXCHANGE" announcing Critical Practice reconfiguration of General Assembly at The Truth is Concrete

Reviewer: ```suggestion
    alt: P2P Exchange event flyer
```

  - src: tic_logo.jpg

>     alt: Poster showing gray crocheted cat figure holding small weapon with orange text "TRUTH IS CONCRETE" and caption about depicting uranium munitions

Reviewer: ```suggestion
    alt: Poster titled Truth is Concrete, showing gray crocheted cat figure with bloodied arm
```
